benefit/data/sheets/CaseEventToFields/CaseEventsToFields-writeAdjournmentNotice.json

```diff
@@ -0,0 +1,332 @@
+    "PageDisplayOrder": 10,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "tribunalDirectPoToAttend",
+    "PageLabel": "Tribunal Direct PO to Attend",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseNextHearingListingDurationType",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 11,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "nextHearingListingDuration",
+    "PageLabel": "How long should the next hearing be listed for?",
+    "PageShowCondition": "adjournCaseTypeOfNextHearing = \"faceToFace\" OR adjournCaseTypeOfNextHearing = \"video\" OR adjournCaseTypeOfNextHearing = \"telephone\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseNextHearingListingDuration",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "adjournCaseNextHearingListingDurationType = \"nonStandardTimeSlot\"",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 11,
+    "PageFieldDisplayOrder": 2,
+    "PageID": "nextHearingListingDuration",
+    "PageLabel": "How long should the next hearing be listed for?",
+    "ShowSummaryChangeOption": "Y",
+    "RetainHiddenValue": "Yes"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseNextHearingListingDurationUnits",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "FieldShowCondition": "adjournCaseNextHearingListingDurationType = \"nonStandardTimeSlot\"",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 11,
+    "PageFieldDisplayOrder": 3,
+    "PageID": "nextHearingListingDuration",
+    "PageLabel": "How long should the next hearing be listed for?",
+    "ShowSummaryChangeOption": "Y",
+    "RetainHiddenValue": "Yes"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseInterpreterRequired",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "OPTIONAL",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 12,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "interpreterRequired",
+    "PageLabel": "Language interpreter",
+    "PageShowCondition": "adjournCaseTypeOfNextHearing = \"faceToFace\" OR adjournCaseTypeOfNextHearing = \"video\" OR adjournCaseTypeOfNextHearing = \"telephone\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseEventID": "adjournCase",
+    "CaseFieldID": "adjournCaseInterpreterLanguageList",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "OPTIONAL",
+    "FieldShowCondition": "adjournCaseInterpreterRequired = \"Yes\"",
+    "LiveFrom": "01/01/2018",
+    "PageDisplayOrder": 12,
+    "PageFieldDisplayOrder": 2,
+    "PageID": "interpreterRequired",
+    "PageLabel": "Language interpreter",
+    "ShowSummaryChangeOption": "Y",
+    "RetainHiddenValue": "Yes"
+  },
+  {
+    "CallBackURLMidEvent": "${CCD_DEF_TRIBUNALS_API_URL}/ccdMidEvent",
```

Comment:
this is the midEvent that we want to create a specific handler, isn't it?
if so, we should follow the same naming convention as they've done for the other midEvent
${CCD_DEF_TRIBUNALS_API_URL}/ccdMidEventAdjournCasePopulateVenueDropdown